operators/src/processing/raster_vector_join/aggregator.rs

```diff
@@ -155,7 +155,7 @@ where
     }
 
     fn add_null(&mut self, feature_idx: usize) {
-        if !self.null[feature_idx] {
+        if self.pristine[feature_idx] {
             self.null[feature_idx] = true;
             self.number_of_non_null_values -= 1;
```

Comment:
Shouldn't this… now… have some effect on the number of pristine values?

Reply:
if I understand the aggregator correctly it takes the first value, regardless of whether it is null or not. So the logic for add_value and add_null would be: if the value is pristine, use it as the aggregation result and mark the feature as not pristine. If it is not pristine: ignore the value. I think the aggregator does exactly that